app/models/ext_management_system.rb

```diff
@@ -45,6 +45,8 @@ def self.supported_types_and_descriptions_hash
   has_many :storages,       -> { distinct },          :through => :hosts
   has_many :ems_events,     -> { order "timestamp" }, :class_name => "EmsEvent",    :foreign_key => "ems_id",
                                                       :inverse_of => :ext_management_system
+  has_many :carried_events, -> { order "timestamp" }, :class_name => "EmsEvent",    :foreign_key => "source_ems_id",
```

Comment:
Yes, sure
This provider brings in events (and other things) the will end up belonging to other providers.
For each one of those alerts we will need to know
- to which ems they belong (ems_events)
- from which ems they came (carried_events)